packages/fluttium/lib/src/tester.dart

```diff
@@ -103,23 +104,23 @@ class Tester {
   ///
   /// The [text] can be a [String] that can also be used as a [RegExp].
   Future<SemanticsNode?> find(String text, {Duration? timeout}) async {
-    var nodes = _findNodes(_semanticsOwner.rootSemanticsNode!, text);
+    var nodes = _findNodes(_semanticsOwner?.rootSemanticsNode, text);
```

Comment:
```suggestion
    var nodes = _findNodes(_semanticsOwner!.rootSemanticsNode, text);
```

I would force non-null here because the semantics need to be available no matter what for Fluttium to work.